options.js

```diff
@@ -535,33 +535,33 @@ document.addEventListener('DOMContentLoaded', function() {
     document.getElementById('remove_user_filter_btn').addEventListener('click', removeUserFilter);
 
     // Generate the menu on the left
-    var menu_list = document.getElementById('menu_list');
-    var headers = document.getElementById('content').getElementsByTagName('h2');
-    for (var i = 0; i < headers.length; i++) {
-        var header = headers[i];
-        var labels = header.getElementsByTagName('label');
-        if (labels.length == 0) {
-            // Add section header to the menu
-            var li = document.createElement('li');
-            li.innerHTML = '<div class="group_header">' + header.innerHTML + '</div>';
-            menu_list.appendChild(li);
-        } else if (labels.length == 1) {
-            var label = labels[0];
-            var scriptName = label.innerHTML.trim();
-            var anchor = 'header' + i;
-
-            // Add anchor to the content section
-            var a = document.createElement('a');
-            a.name = anchor;
-            a.innerHTML = '<br>';
-            content.insertBefore(a, header);
-
-            // Add list item to the menu
-            var li = document.createElement('li');
-            li.innerHTML = '<a href="#' + anchor + '">' + scriptName + '</a>';
-            menu_list.appendChild(li);
-        }
-    }
+    // var menu_list = document.getElementById('menu_list');
+    // var headers = document.getElementById('content').getElementsByTagName('h2');
+    // for (var i = 0; i < headers.length; i++) {
+    //     var header = headers[i];
+    //     var labels = header.getElementsByTagName('label');
+    //     if (labels.length == 0) {
+    //         // Add section header to the menu
+    //         var li = document.createElement('li');
+    //         li.innerHTML = '<div class="group_header">' + header.innerHTML + '</div>';
+    //         menu_list.appendChild(li);
+    //     } else if (labels.length == 1) {
+    //         var label = labels[0];
+    //         var scriptName = label.innerHTML.trim();
+    //         var anchor = 'header' + i;
+
+    //         // Add anchor to the content section
+    //         var a = document.createElement('a');
+    //         a.name = anchor;
+    //         a.innerHTML = '<br>';
+    //         content.insertBefore(a, header);
+
+    //         // Add list item to the menu
+    //         var li = document.createElement('li');
+    //         li.innerHTML = '<a href="#' + anchor + '">' + scriptName + '</a>';
+    //         menu_list.appendChild(li);
+    //     }
+    // }
```

Comment:
Either delete the code or don't, please don't comment this crap out.

Reply:
This PR is intended to be branched upon necessary work is done for cross-compatibility, and that commented code needs to be worked on for cross-compatibility, or perhaps refactored as a whole.

Reply:
Ah I see, are some portions known to be "Chrome safe" and others not?  For the non-Chrome-safe portions, can we just keep both the old and the new code and stick an "if" in there to choose which one to run based on the browser?

Reply:
The issue seems to be that Firefox doesn't like the styling in that pop-up (when called from the overflow/addon bar). The javascript that intends to serve a menu of shortcuts is non-functional and I didn't have time to figure out why.

A better cross-browser approach might be a new tab page with actual in-page tabification for flipping through sections rather than a pop-up.

Reply:
IIRC that code for the menu is some garbage I threw together quickly, it's unclear whether the menu is even useful.  There aren't so many options in ChromeShack that you can't just scroll through them, especially if we moved to a new tab page instead of the small pop-up.